website/project/model.py

```diff
@@ -75,16 +74,9 @@ def has_anonymous_link(node, auth):
     :param str link: any view-only link in the current url
     :return bool anonymous: Whether the node is anonymous to the user or not
     """
-    view_only_link = auth.private_key or request.args.get('view_only', '').strip('/')
```

Comment:
I'm not going to audit `auth.private_key` in `has_anonymous_link` any further, because of the values I assign to these variables:

 - my suspicion that `auth.private_key` ever materially differs from `request.args.get('view_only', '').strip('/')`—**low**
 - the severity of the theoretical bug where it *does* differ—**low**
 - the time it would take to complete such an audit—**high**